etl/steps/data/garden/ihme_gbd/2023-05-15/gbd_mental_health_prevalence_rate.meta.yml

```diff
@@ -0,0 +1,56 @@
+# (Inherited from meadow, remove if not different.)
+tables:
+  gbd_mental_health_prevalence_rate:
+    # (Inherited from meadow, remove if not different.)
+    variables:
+      prevalence_rate_anxiety_disorders:
+        title: Anxiety disorders (prevalence rate)
+        description: >-
+          Prevalence rate of anxiety disorders. This is estimated as the total number of cases with anxiety disorders per 100,000 population.
+        unit: "per 100,000 people"
+      prevalence_rate_bipolar_disorders:
+        title: Bipolar disorders (prevalence rate)
+        description: >-
+          Prevalence rate of bipolar disorders. This is estimated as the total number of cases with anxiety disorders per 100,000 population.
```

Comment:
Shouldn't this be

> Prevalence rate of bipolar disorders. This is estimated as the total number of cases with ~anxiety~ **bipolar** disorders per 100,000 population.

(I guess you could also just leave it out and only keep "number of cases with disorders". It's clear from the first sentence)